html/components/SimpleHtml.js

```diff
@@ -25,6 +25,24 @@ import getEmptyObjectKeys from '../services/getEmptyObjectKeys';
 import isValidVideoFormat from '../services/isValidVideoFormat';
 import Image from './Image';
 
+const UNSPLASH_IMAGE_CONTAINER_ID = 'unsplash-image-container';
```

Comment:
This ID, is it something that the RTE on the Builder generates?

If so, I'm not 100% certain this code is meant to be in the @shoutem/ui lib, because the lib itself is independent from the Builder.

Might be better to expose a prop with which we could then handle this. I leave this to your discretion however.

Reply:
This ID comes from HTML "generated" when inserting Unsplash image - new upcoming feature.
By "generated" I mean I've hardtyped HTML code to be inserted in CMS's Rich Text Editor.

Reply:
Maybe the alternative then is to allow 3rd party developers to turn this alter off?

This might be a separate improvement in and of itself, as we have multiple alters like that, so we may want to give 3rd party developers the chance to override this if we already don't.

For now, let's leave this as it is and revisit this at some point.